docs/_toc.yml

```
@@ -1,16 +1,19 @@
 format: jb-book
-root: README.md
+root: docs/index.md
 chapters:
-- file: docs/statement.md
-- file: docs/community/information.rst
+# - file: docs/statement.md (perhaps keep this on its own tab)
```

This probably does need to be its own file. Accessibility statements typically are on their own page so that people may find them quickly.

```suggestion
- file: docs/statement.md
```